docs/source/using-diffusers/schedulers.mdx

```diff
@@ -0,0 +1,262 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
```

Comment:
@pcuenca @patil-suraj @anton-l - this doc is new. Would be great if you could give it a look